### PR TITLE
A few fixes for 404.md

### PR DESCRIPTION
**Pull request by yasserzamani**

### CODE REVIEW

File: source/404.md [hunk 1]
```diff
@@ -6,16 +6,14 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed.  
-Use menu above to navigate or go to [home page](/).  
-If you noticed any site problem, typos, etc on any page, 
-please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
+The page was moved or never existed. Use menu above to navigate or go to [home page](/).
+If you noticed any site problem (typos, etc on any page), please correct via [pull requesting](https://github.com/apache/struts-site/compare).
```
Comment (paulvi) [inline source/404.md]: https://github.com/apache/struts-site/compare is not a way to start PR, but editing is.

File: source/404.md [hunk 2]
```diff
@@ -6,16 +6,14 @@ title: 404 Page Not Found
 
 # 404 Page Not Found
 
-The page was moved or never existed.  
-Use menu above to navigate or go to [home page](/).  
-If you noticed any site problem, typos, etc on any page, 
-please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
+The page was moved or never existed. You may use menu above to navigate or go to [home page](/).
+If you noticed any site problem (typos, etc on any page), you're able to correct it via [pull requesting](https://github.com/apache/struts-site/compare).
```
Comment (lukaszlenart) [inline source/404.md]: The previous sentence was better (i.e. _`Edit on GitHub` button ..._) also navigating users to the compare is not a good idea, they first must create a branch (which can be done vie `Edit on Github`) to be able create a PR.

**Comment by yasserzamani** [comment]
Ouch. You're right. I didn't see that at home page and 404 and forgot that button existence :) Sorry.

**Comment by lukaszlenart** [comment]
It would be good to add the button there as well :)

**Comment by yasserzamani** [comment]
Nice! I'll see ASAP. Currently am on cell :)

**Pull request closed without merging**